lib/spree/shipment_notice.rb

```diff
@@ -60,7 +60,6 @@ def ship_it!
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
-        shipment.reload.ready! if shipment.pending?
```

Comment:
We still need this, no? If `Spree::Config.shipstation_capture_at_notification` is true, we'll be sending `pending` shipments that we need to `ready!` before we can `ship!`